### DISCUSSION TITLE
Managing Open Source based Project

### DISCUSSION THREAD

**Discussion by plee61** [community/community#115205] [Other Feature Feedback, Questions, & Ideas]
We have a project implemented 2 years ago and its code is based on an open source project downloaded from GitHub. 
The production and development environments for that project are hosted on a cloud server. The latest updates on GitHub has never been downloaded as users do not want some of the new features and also having to deal with merge conflicts.

However, I would like to host the production code to our company GitHub. And I would like to know how do we manage merging the latest code to our production. Hopefully we can also hide/remove those new features users do not want from the production branch.
What is the expectation for keeping the code up to date with the open source code, is it ok if we do not update the based code with the latest?

**Comment by ItayShallev** [comment]
.

**Reply by plee61** [reply]
what was your comment?
